layers/acrn-layer/recipes-vms/targets/acrn-vm-targets_1.0.0.bb

```diff
@@ -0,0 +1,33 @@
+# Copyright (C) 2019 Viktor Sjölind <vsjolind@luxoft.com>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+DESCRIPTION = "Systemd targets for starting ACRN VMs"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=dd98d01d471fac8d8dbdd975229dba03"
+
+RDEPENDS_${PN} = "acrn-vm-init"
+
+inherit systemd
+
+S = "${WORKDIR}"
+
+SRC_URI = "                \
+    file://LICENSE         \
```

Comment:
what do you need LICENSE for?

Reply:
I need the LICENSE to for the yocto licence check. I get that there is not much code to speak of but Yocto does not care and I do not want to put this as "CLOSED" or "COMMERCIAL"